docs/using/polar-syntax.rst

```diff
@@ -285,17 +285,17 @@ will compare the value of the variable age with 10 and unify if it's less than 1
 Cut
 ^^^
 
-The *cut* operator, which in Polar is written as ``cut()``, commits the query
-engine to the enclosing rule definition, and refuses to consider any others. Any
-definitions that have already run are not "un-run", though, or avoided by using
-cut; it just ensures that no *others* will run. Such "other" rule definitions
-are often less specific rules, and the use of `cut()` can be used, e.g., to
-override an ``allow`` rule for a less-specific class.
-
-``cut()`` can appear anywhere in a rule body, but terms that
+The ``cut`` operator the query engine to the enclosing rule definition,
+and refuses to consider any others. Rules that have already run are not
+"un-run", though, or avoided by using cut; it just ensures that no
+*others* will run. Such "other" rule definitions are often less specific
```

Comment:
Would love to add a phrase underscoring _why_ the "other" rules are less specific, e.g.:
> Due to the way in which oso sorts rules for evaluation, such "other" rule definitions...

Reply:
OK, fell into it a little and added more than a phrase.